.github/workflows/main.yml

```diff
@@ -18,22 +18,14 @@ jobs:
           check-latest: true
           cache: npm
 
-      - name: Cache dependencies
-        id: cache-dependencies
-        uses: actions/cache@v3
-        with:
-          path: node_modules
-          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
-
       - name: Install dependencies
-        if: steps.cache-dependencies.outputs.cache-hit != 'true'
```

Comment:
Can you comment on this change too? Are node_modules being cached automatically now or are disabling caching because `npm ci` is fast enough to do from scratch every time?

Reply:
The latter. Looking at tasks that are restoring and populating this cache they take about 22 seconds, this one takes 15. So I removed it as it's not worth the added complexity in the workflow.

Reply:
Should we try caching `~/.npm` instead of `node_modules` [as the docs suggest](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows), to see how that works just in case? In theory the cache store should be smaller and easier to restore.

Reply:
This is already done by providing `cache: npm` to the action that sets up Node.js (see [their docs](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data)).